bridge/github/export.go

```diff
@@ -599,18 +617,14 @@ func (ge *githubExporter) getLabelsIDs(gc *githubv4.Client, repositoryID string,
 }
 
 // create a github issue and return it ID
-func createGithubIssue(gc *githubv4.Client, repositoryID, title, body string) (string, string, error) {
+func createGithubIssue(ctx context.Context, gc *githubv4.Client, repositoryID, title, body string) (string, string, error) {
 	m := &createIssueMutation{}
 	input := githubv4.CreateIssueInput{
 		RepositoryID: repositoryID,
 		Title:        githubv4.String(title),
 		Body:         (*githubv4.String)(&body),
 	}
 
-	parentCtx := context.Background()
```

Comment:
Unless I'm missing something, you still need the timeouts,

Reply:
Agreed, i thought that cancel is enough for this case, which is not correct in case

Reply:
You still need the timeouts.